dm/pkg/terror/error_list.go

```diff
@@ -1071,6 +1072,7 @@ var (
 	ErrSyncerParseDDL                       = New(codeSyncerParseDDL, ClassSyncUnit, ScopeInternal, LevelHigh, "parse DDL: %s", "Please confirm your DDL statement is correct and needed. For TiDB compatible DDL, see https://docs.pingcap.com/tidb/stable/mysql-compatibility#ddl. You can use `handle-error` command to skip or replace the DDL or add a binlog filter rule to ignore it if the DDL is not needed.")
 	ErrSyncerUnsupportedStmt                = New(codeSyncerUnsupportedStmt, ClassSyncUnit, ScopeInternal, LevelHigh, "`%s` statement not supported in %s mode", "")
 	ErrSyncerGetEvent                       = New(codeSyncerGetEvent, ClassSyncUnit, ScopeUpstream, LevelHigh, "get binlog event error: %v", "Please check if the binlog file could be parsed by `mysqlbinlog`.")
+	ErrSyncerGetCharsetMap                  = New(codeSyncerGetCharsetMap, ClassSyncUnit, ScopeUpstream, LevelHigh, "get upstream charset and default collation error", "Please check if upstream is normal.")
```

Comment:
```suggestion
	ErrSyncerGetCharsetMap                  = New(codeSyncerGetCharsetMap, ClassSyncUnit, ScopeUpstream, LevelHigh, "retrieve upstream charset and default collation error", "Please check if upstream is normal.")
```

Prefer to use `retrieve` instead of `get`, see: https://wikidiff.com/retrieve/get

Reply:
ok, it is better.